src/main/java/edu/wpi/first/shuffleboard/Shuffleboard.java

```diff
@@ -40,6 +41,10 @@ public void init() throws Exception {
   @Override
   public void start(Stage primaryStage) throws IOException {
     primaryStage.setScene(new Scene(mainPane));
+    primaryStage.setMinWidth(640);
+    primaryStage.setMinHeight(480);
+    primaryStage.setWidth(Screen.getPrimary().getBounds().getWidth());
+    primaryStage.setHeight(Screen.getPrimary().getBounds().getHeight());
```

Comment:
should be getVisualBounds per https://github.com/wpilibsuite/shuffleboard/pull/55#issuecomment-318926171